Services/UI/OJS.Services.Ui.Data/Implementations/SubmissionsForProcessingDataService.cs

```diff
@@ -50,20 +63,42 @@ public async Task AddOrUpdateBySubmissionIds(ICollection<int> submissionIds)
             }
         }
 
-        public async Task AddOrUpdateBySubmission(int submissionId)
+        public async Task AddOrUpdateReprocessingBySubmission(int submissionId)
         {
             var submissionForProcessing = await this.GetBySubmission(submissionId);
 
             if (submissionForProcessing != null)
             {
-                submissionForProcessing.Processing = false;
+                submissionForProcessing.Processing = true;
```

Comment:
By default the submission should be Processing = false. Only when it is queued should be set to Processing = true

Reply:
Done